tests/report_tests/plugin_tests/teamd.py

```diff
@@ -25,6 +25,9 @@ class TeamdPluginTest(StageTwoReportTest):
     sos_cmd = '-o teamd'
     redhat_only = True
 
+    # teaming has been deprecated from RHEL 9
+    versions = ['8']
```

Comment:
Nit: `os_versions` or `only_os_versions` would be more descriptive here.

Reply:
Yes, it sounds much better this way. I'll fix and push again, thank you!